main.py

```diff
@@ -1,12 +1,209 @@
-import requests
+from check_in_check_out_list import CheckInOut
+from requests.api import options
+from video_list import VideoList
+from customer_list import CustomerList
 
-URL = "http://127.0.0.1:5000"
-BACKUP_URL = "https://retro-video-store-api.herokuapp.com"
+# URL = "http://127.0.0.1:5000"
 
-def main():
-    print("WELCOME TO RETRO VIDEO STORE")
-    pass
+# BACKUP_URL = "https://halisas-retro-video-store-api.herokuapp.com"
+
+def main(play=True):
+
+    video_store = VideoList(url="http://localhost:5000/")
+    customer_store = CustomerList(url="http://localhost:5000/")
+    rental_store = CheckInOut(url="http://localhost:5000/")
```

Comment:
A couple of notes here:

1.  Each of these classes are doing the **same** thing.  Each one is acting as an intermediary between `main.py` and the API.  So you could condense them down to 1 class. 
2. The assignment does have you use the provided heroku api.